documentation/0.13.0/html/proxy-quick-start/index.markdown

```diff
@@ -4,11 +4,11 @@ title: Proxy quick start
 version: 0.13.0
 ---
 
-This quick start will guide you through deploying the proxy.
+This quick start will guide you through deploying the proxy, either locally on your machine, or in a container via Docker Compose.
```

Comment:
I'm thinking ultimately this ought to be a standalone quickstart.  proxy-quick-start should be renamed to something like 'quickstart-standalone' and this would be 'quickstart-docker'.   I'm more thinking out loud here, rather than asking for changes to this particular PR.

You might have noticed that we are putting quite a lot of energy into revamping the website right now and part of that is getting the quickstart experience right.